src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -98,12 +98,8 @@ public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $q
             if ($this->isPropertyNested($property)) {
```

Comment:
@dunglas in fact this might call the `addJoinsForNestedProperties` method from the `AbstractFilter`, should I change this as well? We can then set `private` on `addJoinOnce`.